src/components/Taxonomy/ActiveTopicConnection.tsx

```diff
@@ -34,62 +28,78 @@ const StyledFlexWrapper = styled.div`
   align-items: center;
 `;
 
+const StyledPrimaryConnectionButton = styled.button`
+  border: 0;
+  border-radius: ${misc.borderRadius};
+  padding: ${spacing.xsmall} ${spacing.small};
+  margin-right: ${spacing.xsmall};
+  text-transform: uppercase;
+  ${fonts.sizes(14, 1.1)};
+  font-weight: ${fonts.weight.semibold};
+  background: ${colors.support.green};
+  opacity: 0.3;
+  transition: opacity 100ms ease;
+  &:hover,
+  &:focus {
+    opacity: 1;
+  }
+  &[data-primary='true'] {
+    opacity: 1;
+  }
+`;
+
+const StyledConnections = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: ${colors.brand.greyLightest};
+  padding: ${spacing.xsmall};
+  margin-bottom: 2px;
```

Comment:
Bruke verdi fra spacing?

Reply:
2px finnes ikke, og er ikke ekstremt fan av `${spacingUnit / 12}px`

Reply:
Ok, så det må være akkurat 2px? isåfall er det jo bare å beholde ja